src/qt/trafficgraphwidget.cpp

```diff
@@ -22,8 +27,17 @@ TrafficGraphWidget::TrafficGraphWidget(QWidget *parent) :
     timer(0),
     fMax(0.0f),
     nMins(0),
-    vSamplesIn(),
-    vSamplesOut(),
+    nSamplesWindowSize(0),
+    vSamplesIn(MAX_SAMPLES),
+    vSamplesOut(MAX_SAMPLES),
+	
+	//MLUMIN 05_22: Another vector set for 'display samples' vs data samples, and what's on-display vs off-display.
+    vSamplesToDisplayIn(DESIRED_SAMPLES), 
+    vSamplesToDisplayOut(DESIRED_SAMPLES),
+    nSamplesHeadIndex(0),
+    nSamplesTailIndex(0),
+    nSamplesChunkSize(0),
+	
     nLastBytesIn(0),
     nLastBytesOut(0),
     clientModel(0)
```

Comment:
This throws a bunch of compiler warnings because the initialization is done in a different order from the class declaration in the header file. Easy fix, just use the same order:

```suggestion
    vSamplesIn(MAX_SAMPLES),
    vSamplesOut(MAX_SAMPLES),

    vSamplesToDisplayIn(DESIRED_SAMPLES),
    vSamplesToDisplayOut(DESIRED_SAMPLES),

    nLastBytesIn(0),
    nLastBytesOut(0),
    clientModel(0),

    nSamplesWindowSize(0),
    nSamplesHeadIndex(0),
    nSamplesTailIndex(0),
    nSamplesChunkSize(0)
```

Reply:
Roger. Will fix.